### DISCUSSION TITLE
Any workarounds for merging?

### DISCUSSION THREAD

**Discussion by sheluchin** [jelmer/dulwich#937] [General]
Is there any temporary workaround we can use to achieve merging while official support for merging is being worked out?

Related to https://github.com/dulwich/dulwich/issues/452

**Comment by jelmer** [comment]
Stash has a git-merge implementation that's based on Dulwich, but I have never used it. See https://github.com/ywangd/stash/blob/master/lib/git/git-merge.py

The other alternative is to invoke C Git using subprocess.

**Comment by jelmer** [comment]
FWIW merging is now properly supported.
